test/test_examples_2d.jl

```diff
@@ -5,310 +5,31 @@ using Trixi
-      linf = [4.667437348149228e-5, 0.03175420871507906, 0.0318039789241531, 0.04561735256198318])
-  end
+  include("test_examples_2d_euler.jl")
 
 
   # MHD
-  @testset "elixir_mhd_alfven_wave.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave.jl"),
-      l2   = [0.00011134327871126106, 5.880012991229279e-6, 5.880012991198096e-6, 8.432976937044043e-6, 1.294412985383592e-6, 1.2239229634469704e-6, 1.2239229634550153e-6, 1.8308368268137684e-6, 8.098531574321276e-7],
-      linf = [0.0002678911590654476, 1.6247294531021583e-5, 1.6247294531257506e-5, 2.7353154099893362e-5, 5.328011688177092e-6, 8.115802777819425e-6, 8.115802777708403e-6, 1.2104880218147263e-5, 4.179631827493849e-6])
-  end
-
-  @testset "elixir_mhd_alfven_wave_mortar.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave_mortar.jl"),
-      l2   = [4.610806814382062e-6, 1.6898720654838787e-6, 1.6209422082463377e-6, 1.6995459258068932e-6, 1.486433605116548e-6, 1.3876687274336836e-6, 1.3412591287808953e-6, 1.715595049725351e-6, 9.813856592499105e-7],
-      linf = [3.5225644818837054e-5, 1.5350075873457603e-5, 1.4264708736805298e-5, 1.4421606211581506e-5, 7.744178943891455e-6, 1.0188298347757474e-5, 9.862369056090614e-6, 1.6018083168742314e-5, 5.563792310153545e-6],
-      tspan = (0.0, 1.0))
-  end
-
-  @testset "elixir_mhd_ec.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec.jl"),
-      l2   = [0.03628213711730772, 0.043011106541313036, 0.04299694609483342, 0.025746634517134097, 0.1620505609687871, 0.01745407144590488, 0.017454880786441977, 0.026880815662339404, 0.0001427849600999494],
-      linf = [0.2350909734343778, 0.31557861254908504, 0.30933432081715095, 0.21172448318061543, 0.9738530715930489, 0.09072863714076829, 0.09150557090076483, 0.1572617079252614, 0.004407940752443958])
-  end
-
-  @testset "elixir_mhd_orszag_tang.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_orszag_tang.jl"),
-      l2   = [0.21672079513239845, 0.263644834080207, 0.31388290886723225, 0.0, 0.5122895724585859, 0.22917747391916293, 0.3430662443376359, 0.0, 0.0031626307824603],
-      linf = [1.2643446538250775, 0.6735720190596391, 0.8584758183037491, 0.0, 2.8094485872574957, 0.6587140271779941, 0.9599314012066199, 0.0, 0.052797557445107016],
-      tspan = (0.0, 0.09))
-  end
-
-  @testset "elixir_mhd_orszag_tang.jl with flux_hll" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_orszag_tang.jl"),
-      l2   = [0.10794984293584534, 0.20177076633535806, 0.22968340939376386, 0.0, 0.29942323222232975, 0.156948176949458, 0.24306814332384577, 0.0, 0.013279172422637332],
-      linf = [0.5601603816370189, 0.5107264221998954, 0.6568655409846257, 0.0, 0.9903131924303785, 0.3993471648741348, 0.6835807774807094, 0.0, 0.520606845234404],
-      tspan = (0.0, 0.06), surface_flux = flux_hll)
-  end
+  include("test_examples_2d_mhd.jl")
```

Comment:
Why not also split the tests for the initial conditions?

Reply:
Because I don't want to have too many files lying around. I can understand the argument "I want to have a quick check whether I broke some system fundamentally" - it should be enough to run the basic example tests for this, I think.

Reply:
:+1: